docs/messaging/messaging-spans.md

```diff
@@ -352,6 +352,18 @@ different processes could be listening on TCP port 12345 and UDP port 12345.
 | `receive` | One or more messages are requested by a consumer. This operation refers to pull-based scenarios, where consumers explicitly call methods of messaging SDKs to receive messages. |
 | `deliver` | One or more messages are passed to a consumer. This operation refers to push-based scenarios, where consumer register callbacks which get called by messaging SDKs. |
 
+`messaging.system` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+
+| Value  | Description |
+|---|---|
+| `activemq` | Apache ActiveMQ |
+| `amazon_sqs` | Amazon Simple Queue Service (SQS) |
```

Comment:
Interesting, the product name seems to actually be "Amazon SQS", but using the convention from cloud.product, wouldn't aws_sqs be more fitting? Also aligned with gcp_pubsub.